src/MembershipManager.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types = 1);
```

Comment:
Curious to see if this will break any alpha blockers.

Reply:
Me too, that's why I added it, in the current code base all tests are passing with strict types! :+1: